ngraph/test/util/engine/ie_engines.cpp

```diff
@@ -87,12 +170,6 @@ namespace
                                                   const size_t tolerance_bits)
     {
         const auto& computed_precision = computed->getTensorDesc().getPrecision();
-        const auto& expected_precision = expected->getTensorDesc().getPrecision();
-
-        if (computed_precision != expected_precision)
```

Comment:
Why did you remove this check?

Reply:
When we work with a data type that converts to another, for example U16, we get computed_precision = I32 and expected_precision = U16.